content/concept-issuing-and-operational-addresses.md

```diff
@@ -8,9 +8,9 @@ All non-XRP currency balances (issuances) in the Ripple Consensus Ledger (RCL) a
 
 [![Diagram: Funds flow from the issuing address to standby addresses, to operational addresses, to customer and partner addresses, and finally back to the issuing address.](img/funds_flow_diagram.png)](img/funds_flow_diagram.png)
 
-When the issuing address sends payments, it creates balances in the accounting relationships in the Ripple Consensus Ledger. Within the RCL, users can exchange balances across different accounting relationships, so we use the term _issuances_ to describe any non-XRP balance. Issuances have positive value from the perspective of the customers, partners, operational addresses, and standby addresses. The same issuances are negative balances from the perspective of the issuing address, since they represent obligations. When the issuing address receives a payment, this reduces its obligations, effectively "destroying" the issuances that were sent.
+When the issuing address sends payments, it creates balances in the accounting relationships in the Ripple Consensus Ledger. Within the RCL, users can exchange balances across different accounting relationships, so we use the term _issuances_ to describe any non-XRP balance. Issuances have positive value from the perspective of the counterparties. The same issuances are negative balances from the perspective of the issuing address, since they represent obligations. When the issuing address receives a payment, this reduces its obligations, erasing the issuances that were sent.
```

Comment:
"Issuances have positive value from the perspective of the counterparties."

For ideal clarity, maybe specify which counterparties, i.e. "...the issuing address's counterparties." 

And to emphasize the relationship between the perspectives, perhaps make the construction of the next sentence a little more parallel to the previous one, i.e. "Issuances have negative value from the perspective of the issuing address..."